operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -95,11 +100,12 @@ protected Status createStatus() {
                 .onComplete(context.succeeding(v -> context.verify(() -> {
                     MeterRegistry registry = metrics.meterRegistry();
 
+                    assertThat(registry.getMeters().get(0).getId().getTags().size(), is(2));
+                    assertThat(registry.getMeters().get(0).getId().getTags().get(1), is(Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")));
```

Comment:
This tests the label only for the first metric? What metric is that? Can't you get it by name? That would probably be more stable.

Reply:
assertThat(registry.find(AbstractOperator.METRICS_PREFIX + "reconciliations").meter().getId().getTags().size(), is(2));
Will something like this work @scholzj ?

Reply:
That is more like what I had in mind, yes. All the other asserts seem to be using `get` and not `find`. So maybe you can do the same?

Reply:
Yeah, Iam changing them